cmd/errors.go

```diff
@@ -54,7 +54,7 @@ remove that operation from the "pending_operations" section of the file. Once th
 use 'pulumi stack import' to import the repaired stack.`)
 	contract.IgnoreError(writer.Flush())
 
-	cmdutil.Diag().Errorf(diag.RawMessage("" /*urn*/, buf.String()))
+	cmdutil.Diag().Errorf(diag.RawMessage("" /*urn*/, buf.String()), false)
```

Comment:
nit: like with `/*urn*/` it can be nice to have these named-arg-comments that help explain what this param is for.